docs/setup.md

```diff
@@ -8,47 +8,80 @@ description: |-
 
 # Databricks-Sync
 
-Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+Databricks-Sync is a synchronization utility for the export and import of Databricks-native objects from one workspace to another. The primary use cases for Databricks-Sync are Disaster Recovery, migration, and CI/CD.
 
 ## Prerequisites
 
-Visit the [prerequisites page](prerequisites.md) to check if you are missing any prerequisites before beginning installation.
+### Databricks CLI
+
+Databricks-Sync authenticates to Databricks via the Databricks CLI. The Databricks CLI should be installed and configured to authenticate to the Databricks workspace using Personal Access Tokens. For detailed instructions to install and configure the Databricks CLI, please view the official [Databricks documentation](https://docs.databricks.com/dev-tools/cli/index.html#databricks-cli). It is a best practice to set up a unique profile within the Databricks CLI for each workspace.
+
+Check your Databricks CLI access credentials in the file `~/.databrickscfg` to verify successful authentication set up. The file should contain entries like:
```

Comment:
@stikkireddy should we ask them to run databricks command to check it or look into this file?

Reply:
Databricks docs for the CLI verifies by looking into the file as well

Reply:
I'm god with that :)